core/client/fs/src/main/java/alluxio/client/block/stream/BlockInStream.java

```diff
@@ -327,11 +340,54 @@ public void seek(long pos) throws IOException {
     if (pos == mPos) {
       return;
     }
+    // Protect the original seek logic under fuse flag to minimize and isolate
```

Comment:
I think this comment is confusing? The flag is about using the shared grpc reader, so this comment is talking about a fuse flag.

Reply:
clarified.